src/cswinrt/strings/WinRT.cs

```diff
@@ -47,16 +47,6 @@ internal class Platform
         [DllImport("kernel32.dll", SetLastError = true, BestFitMapping = false)]
         internal static extern IntPtr GetProcAddress(IntPtr moduleHandle, [MarshalAs(UnmanagedType.LPStr)] string functionName);
 
-        internal static T GetProcAddress<T>(IntPtr moduleHandle)
```

Comment:
Did a build of your branch.  There are still references to this in other parts of the code base like here: https://github.com/microsoft/CsWinRT/blob/75ffe544984b77c2792f36bde07a3cfd6a5c4656/src/WinRT.Runtime/ExceptionHelpers.cs#L44 .  Might want to restore it.

Reply:
I added that back. Maybe I didn't push yet.